loony_dev/web/services.py

```
@@ -1206,20 +1292,107 @@ def _safe_repo_log_path(base_dir: Path, owner: str, repo: str) -> Path:
     return candidate
 
 
+# Block size for the reverse tail reader. The reader walks the file backward in
+# fixed-size blocks from EOF, so memory + IO are bounded by ``lines × avg-line``
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)